screen/accounts/build.gradle.kts

```diff
@@ -1,9 +1,11 @@
 plugins {
     id("ivy.feature")
+    alias(libs.plugins.screenshot)
 }
 
 android {
     namespace = "com.ivy.accounts"
+    experimentalProperties["android.experimental.enableScreenshotTest"] = true
```

Comment:
Extract this logic as a convention plugin in `buildSrc` like we had for Paparazzi and apply this plugin by default to `ivy.feature`

Reply:
Will update it. 

> @isaacnguyen0809 just to confirm, we don't use Git LFS, right? Because it wants us to pay
Yes, we don't use Git LFS